docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -61,6 +61,10 @@ The module `org.keycloak:keycloak-model-legacy` module was deprecated in a previ
 
 The old behavior to preload offline sessions at startup is now removed after it has been deprecated in the previous release.
 
+= kcadm Changes
+
+How kcadm parses and handles options and parameters has changed. Error messages from usage errors, the wronge option or parameter, may be slightly different than previous versions. Also usage errors will have an exit code of 2 instead of 1.
```

Comment:
```suggestion
How kcadm parses and handles options and parameters has changed. Error messages from usage errors, the wrong option or parameter, may be slightly different than previous versions. Also usage errors will have an exit code of 2 instead of 1.
```
@shawkins I believe it's a typo, right?

Reply:
Correct